text/numbers.talon

```diff
@@ -1,3 +1,3 @@
-not tag: user.mouse_grid_showing
+tag: user.mouse_grid_showing
```

Comment:
This change seems quite incorrect 🤔

Reply:
Oh wow nice catch 😅

Reply:
Woops, lemme go and fix that. Thanks for catching that.

Reply:
Oh, yes, I see when I broke that.

Reply:
Fixed in 1.4.4.

Reply:
Assuming there's now a test for this case? Would probably be good to avoid this kind of regression going forward 😅

Reply:
Ah I see https://github.com/wenkokke/talonfmt/blob/dev/tests/data/golden/simple/default/knausj_talon_text_numbers.yml; looks like that should do it 👍